travvy/src/components/OnboardingPage.vue

```diff
@@ -0,0 +1,154 @@
+<template>  
+  <div class="Onboarding">
+     <div class="container">
+    <!-- Onboarding fields that the user must fill in in order to get personalized recommendations --> 
+    <h2>Welcome to Travvy, we'd love to learn more about you!</h2>
+    <p>Please fill in the following information in order to get a custom travel experience.</p>
+    <label>Please select the sex you identify with: </label>
+    <br>
+    <br>
+    <div class="custom-select">
+    <select>
+        <option></option>
+        <option>Male</option>
+        <option>Female</option>
+        <option>Prefer not to say</option>
```

Comment:
Maybe we should give another option for people that do not fit into male/female? They might be willing to identify but just not fit into the categories (non-binary, etc.)